nata/weather-widget/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Weather</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section class="top-banner">
+      <div class="container">
+        <h1 class="heading">Simple Weather App</h1>
+        <form>
+          <input type="text" placeholder="Search for a city" autofocus />
```

Comment:
Consider using https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search
```suggestion
          <input type="search" placeholder="Search for a city" autofocus />
```